buf/registry/owner/v1/user_service.proto

```diff
@@ -128,7 +130,10 @@ message CreateUsersRequest {
     // The type of the User.
     //
     // If not set, the default USER_TYPE_STANDARD is used.
-    UserType type = 2 [(buf.validate.field).enum.defined_only = true];
+    UserType type = 2 [
+      (buf.validate.field).enum.defined_only = true,
+      (buf.validate.field).enum.not_in = 0
```

Comment:
Doesn't this effectively make it required, given that the field is by default the zero value?

Reply:
A bit more restrictive since you could pass 999 with just required, but defined_only limits it to the defined values.

Reply:
I think the observation here is that if we add this validation, the field is changing from effectively optional (there is a defined default behavior if you pass the zero value) to required (you must provide a value that is not the zero value). 

Does not setting a value for a non-optional enum send the `0` value over the wire?

Reply:
Double checked everything, I think we do not want to restrict `not_in: 0` here since we do actually want to allow `_UNSPECIFIED` to be passed and we use the fallback condition outlined. Going to remove this.

Reply:
Doesn't this call into question the use of not_in altogether?

I had not set it because of the requiredness it enforces.

Reply:
For a singular enum field where unspecified is valid, you don't want it. Elsewhere for repeated enum fields, it doesn't make sense to have unspecified as a valid option, so not_in is appropriate.